desc/magnetic_fields/_core.py

```diff
@@ -886,6 +1081,45 @@ def compute_magnetic_field(
 
         return B
 
+    def compute_magnetic_vector_potential(
+        self, coords, params=None, basis="rpz", source_grid=None
```

Comment:
should this also take `transforms` for consistency with `compute_magnetic_field`? (and in many other places as well)

Reply:
ah yes, good catch. Must've added this before I added transforms everywhere else for compute magnetic field